server/pbenchinacan/container-build.sh

```diff
@@ -63,6 +63,10 @@ buildah run $container rm -f /tmp/pbench-server.rpm
 buildah copy --chown root:root --chmod 0644 $container \
     ${PBINC_SERVER}/lib/config/nginx.conf /etc/nginx/nginx.conf
 
+# Add our Pbench Server CA certificate.
+buildah copy --chown root:root --chmod 0444 $container \
+    ${PBINC_INACAN}/etc/pki/tls/certs/pbench_CA.crt /etc/pki/tls/certs/pbench_CA.crt
+
```

Comment:
Don't we user `container-build.sh` both for the functional tests *and* for the "real" containers we'll eventually deploy to staging/production? I'd really prefer not to be packaging our fake CA in staging much less production even though we need it for functional tests. This probably needs to be conditionalized somehow.

Reply:
Yeah, Dave, we do (and I think _should_) use the same container image for functional testing and for deployment, but I concluded that adding the CA to the container image is the least of evils.

In the Staging/Production case, it's just another file, and a small one at that (1.3Kb).  It's not a private key or anything, it's just a cert.  We would give it away to anyone who wants to verify the TLS connection to a development server, so I'm not concerned about having it in the container.

And, I can't think of any way of conditionalizing this which doesn't impact Staging and Production.

Reply:
Yeah, you have a point. I am not sure what is the best way forward, mounting it or copying it at the build time. @webbnh do you want to weigh in?

Reply:
> @webbnh do you want to weigh in?

[Yes](https://github.com/distributed-system-analysis/pbench/pull/3427#discussion_r1246929719).  😁

Reply:
I really don't like putting our development fake CA bundle into the product container. It may be expedient, but it's wrong and there's got to be a better way.

Reply:
We use the config file, which we map in already, to decide whether to use the private CA or not.  (It's not _fake_....)  If we map it in, then we cannot use the same command/script in both development and staging/production (or, at least, we have to add conditionalization such that the container invocation will be (slightly) different in the two cases).

Instead, for the cost of a tiny increment in size, we avoid the need for conditionalization or divergence in the invocations.